### ISSUE TITLE
bug: routerDirection: 'none', routeAction: 'replace' causes page transition.

### ISSUE THREAD

**Issue by bastischulz** [ionic-team/ionic-framework#23453]
TLDR: As soon as i do something like this: `push(url, 'none', 'replace')`, it results in a 'forward' page transition.

"@ionic/core": "^5.6.9",
"@ionic/react": "^5.6.9",
"@ionic/react-router": "^5.6.9",

I built an url-synced segment, which came along with the following two problems:
1st: I need to prevent the default page transition.
2nd: The back button should link to the actual previous page.

In my understanding,  routerDirection: 'none' should prevent the transition, which it acutally does on its own. But as soon as I try to fix the 2nd problem with routeAction: 'replace', the Back-Button works as intended, but the page-transition is back.

**Comment by liamdebeasi** [comment]
Thanks for the issue! This issue is being closed due to the lack of a code reproduction. If this is still an issue with the latest version of Ionic, please create a new issue and ensure the template is fully filled out.

Thank you for using Ionic!
